src/proxy-ui-api/frontend/src/store/modules/general.ts

```diff
@@ -82,8 +82,9 @@ export const actions: ActionTree<State, RootState> = {
   },
 
   fetchMemberName({ commit }, { memberClass, memberCode }) {
+    // this is currently an inline schema and is not automatically generated to a typescript type
     return api
-      .get(
+      .get<{ member_name: string }>(
```

Comment:
All OpenAPI objects now get generated into TypeScript types. For example this one is `MemberName`.

This was done in PR (already merged in develop): https://github.com/nordic-institute/X-Road/pull/563/files

Reply:
Fixed this